bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -127,11 +127,11 @@ spec:
             secretName: linuxptp-daemon-secret
         {{ if (eq .EnableEventPublisher true) }}
         - name: pubsubstore
-          {{ if not (hasPrefix "amqp" .EventTransportHost) }}
-          persistentVolumeClaim:
-            claimName: cloud-event-proxy-store
-          {{ else }}
+          {{ if or (eq .StorageType "emptyDir") (hasPrefix "amqp" .EventTransportHost) }}
           emptyDir: {}
+          {{ else }}
+          persistentVolumeClaim:
+            claimName: cloud-event-proxy-store-{{ .StorageType }}
```

Comment:
use cloud-event-proxy-store-{{.NodeName}}-{{ .StorageType }} to support PVC for Multi node.
Use similar  to event service